airbyte-integrations/connectors/source-nasa-apod/source_nasa_apod/source.py

```diff
@@ -0,0 +1,124 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from datetime import datetime, time, timedelta
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple, Union
+
+import requests
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+
+date_format = "%Y-%m-%d"
+
+
+class NasaApodStream(HttpStream):
+
+    api_key = "api_key"
+    url_base = "https://api.nasa.gov/"
+
+    primary_key = "date"
+
+    def __init__(self, config: Mapping[str, any], **kwargs):
+        super().__init__()
+        self.config = config
+
+    def path(
+        self, stream_state: Mapping[str, Any] = None, stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> str:
+        return "planetary/apod"
+
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        return None
+
+    def request_params(
+        self, stream_state: Mapping[str, Any], stream_slice: Mapping[str, any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> MutableMapping[str, Any]:
+        return self.config
+
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+        yield response.json()
```

Comment:
```suggestion
        yield from response.json()
```
My local tests are not working without this correction.

Reply:
Could you please provide the `config.json` that you are using?

I think the best approach here is to check if the response corresponds to a `dict` or to a `list`, and then `yield` or `yield from` accordingly.

Reply:
I used one with start date and not a date.